iotedgedev/envvars.py

```diff
@@ -220,8 +213,7 @@ def save_envvar(self, key, value):
             dotenv_path = os.path.join(os.getcwd(), dotenv_file)
             set_key(dotenv_path, key, value)
         except Exception:
-            self.output.error(f("Could not update the environment variable {key} in file {dotenv_path}"))
-            sys.exit(-1)
+            raise IOError(f("Could not update the environment variable {key} in file {dotenv_path}"))
```

Comment:
I prefer consistency, as in the other raised exceptions in this module have normal python string formatting, whereas this one uses the 'fstrings...sorta library'......consider altering this one to the same format as the others.